src/blog/2023/09/modernize-your-legacy-industrial-data-part2.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Modernize your legacy industrial data. Part 2.
+subtitle: A deeper dive with into making the most of legacy industrial data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
+description: A deeper dive with into making the most of legacy industrial data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
+date: 2023-09-17
+authors: ["stephen-mclaughlin"]
+image: "/blog/2023/09/images/industrial-legacy-data-blog-image.png"
+tags:
+    - posts
+    - node-red
+    - community
+---
+
+In an article published recently, I introduced the topic of working with legacy industrial data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world. 
```

Comment:
Let's link the previous one here and remove the line just before the more fold 🙂

Reply:
Sorry ZJ, I am not entirely certain what you are asking here. Could you clarify or suggest please ZJ? 
_(PS, in case you didn't spot it, there is a link to the previous article in this section already)_

Reply:
```suggestion
In [part 1 of this series](modernize-your-legacy-industrial-data-part2.md) for the introduction and overview.) published recently, I introduced the topic of working with legacy industrial data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world. 
```

And remove line 19? I feel like 19 breaks the flow of the article, while we can link in this line